1e/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d86565ae-ca60-45c2-a0b9-944ec1e05a15",
+  "app_id": "one-e",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your 1E product metrics to Datadog.",
+    "title": "1E Integration",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "1e",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "onee.",
+        "metadata_path": "metadata.csv",
+        "check": "onee.experience.score"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "1E Dashboard": "assets/dashboards/1EDashboard.json"
+    },
```

Comment:
```suggestion
    }
```

Reply:
I believe this may be causing the validation checks to fail. Could you try removing to see if that will resolve the issue?